netcipher-webkit/src/info/guardianproject/netcipher/webkit/WebkitProxy.java

```diff
@@ -381,10 +381,13 @@ public static boolean resetLollipopProxy(String appClass, Context appContext) {
     @TargetApi(21) // for android.util.ArrayMap methods
     @SuppressWarnings("rawtypes")
     private static boolean setWebkitProxyLollipop(Context appContext, String host, int port) {
-        System.setProperty("http.proxyHost", host);
-        System.setProperty("http.proxyPort", Integer.toString(port));
-        System.setProperty("https.proxyHost", host);
-        System.setProperty("https.proxyPort", Integer.toString(port));
+        if (host != null) {
+            System.setProperty("http.proxyHost", host);
+            System.setProperty("http.proxyPort", Integer.toString(port));
+            System.setProperty("https.proxyHost", host);
+            System.setProperty("https.proxyPort", Integer.toString(port));
+        }
```

Comment:
I think `resetProxy` (submethod: `resetLollipopProxy`) works by resetting proxy parameters to _null_. So won't this render disabling proxying ineffective to some extend?

Reply:
To answer my own question: no, because_resetProxy_ does the resetting, before calling `setWebkitProxyLollipop(appContext, null, 0);`

---

Actually this block is redundant and should be safe to delete, because _setProxy_ always calls `setSystemProperties` which sets exactly the same values. [WebkitProxy.java#L60](https://github.com/guardianproject/NetCipher/blob/4f0758bfca2e68ce1ecba8f88fa4ea0551340dde/netcipher-webkit/src/info/guardianproject/netcipher/webkit/WebkitProxy.java#L60)

Reply:
It is redundant, but this just makes it consistent with what's in `resetKitKatProxy`.  If you want, I can push a commit removing it from both places.

Reply:
Totally fine in the context of this PR. :+1:

Reply:
Done in 3c67c95